src/java.desktop/windows/classes/sun/awt/windows/WPathGraphics.java

```diff
@@ -95,11 +95,8 @@ final class WPathGraphics extends PathGraphics {
     private static boolean useGDITextLayout = true;
     private static boolean preferGDITextLayout = false;
     static {
-        @SuppressWarnings("removal")
-        String textLayoutStr =
-            java.security.AccessController.doPrivileged(
-                   new sun.security.action.GetPropertyAction(
-                         "sun.java2d.print.enableGDITextLayout"));
+        String textLayoutStr = System.getProperty(
+                     "sun.java2d.print.enableGDITextLayout");
 
         if (textLayoutStr != null) {
             useGDITextLayout = Boolean.getBoolean(textLayoutStr);
```

Comment:
Noticed a possible pre-existing bug:

`Boolean::getBoolean` takes a system property to look up and evaluate to boolean. Here, this method is given a string which is the result of reading the system propery "sun.java2d.print.enableGDITextLayout".

Reply:
@eirbjo Thank you for submitting [JDK-8345934](https://bugs.openjdk.org/browse/JDK-8345934).

[I found this issue](https://github.com/openjdk/jdk/pull/22083#discussion_r1846889627) too, yet it's marked as resolved… because it's out of scope for this change.